src/components/Radio/Radio.tsx

```diff
@@ -31,13 +35,16 @@ export interface IRadioProps {
   onChange?: ComponentEventHandler<IRadioProps>
   type?: string
   styles?: IComponentPartStylesInput
+
+  // TODO: what is the difference between value and checked?
   value?: string | number
```

Comment:
Multiple radio buttons in a group may have different values, we must have this in order to know which radio button is selected. The checked only indicates if this radio button is selected or not.